alignparse/targets.py

```diff
@@ -393,6 +398,10 @@ def __init__(self, *, seqsfile, feature_parse_specs,
                             'mutation_op_count']
 
         # get targets from seqsfile
+        if select_target_names is not None:
+            if len(select_target_names) < 1:
```

Comment:
I think a string would pass this filter, but could pose problems below. Ideally people wouldn't try to use a string as the docs say it must be 'None' or a list. Does specifying the type in the doc string enforce that it has to be a list? If not, it might be good if our check could only be passed by a list?